src/test/java/org/folio/rest/support/HttpClient.java

```diff
@@ -50,17 +59,22 @@ public void post(
     Handler<HttpClientResponse> responseHandler) {
 
     HttpClientRequest request = client.postAbs(url.toString(), responseHandler);
-    request.headers().add("Content-type","application/json");
-    stdHeaders(request, url, tenantId);
+    request.putHeader(CONTENT_TYPE, APPLICATION_JSON);
+    initDefaultHeaders(request, url, tenantId);
 
     if (body == null) {
       request.end();
-      return;
+    } else {
```

Comment:
To me, the null check is effectively a guard clause. Why does this second aspect needs to be wrapped in an else statement?

Reply:
The main reason is to avoid return statement, because it's redundant in a void method